config/config.go

```diff
@@ -388,6 +393,13 @@ type Configuration struct {
 	WorkerExecutionMode ExecutionMode                `json:"worker_execution_mode" envconfig:"CONVOY_WORKER_EXECUTION_MODE"`
 	MaxRetrySeconds     uint64                       `json:"max_retry_seconds,omitempty" envconfig:"CONVOY_MAX_RETRY_SECONDS"`
 	LicenseKey          string                       `json:"license_key" envconfig:"CONVOY_LICENSE_KEY"`
+	Dispatcher          DispatcherConfiguration      `json:"dispatcher"`
+}
+
+type DispatcherConfiguration struct {
+	EnforceSecure bool     `json:"enforce_secure" envconfig:"CONVOY_DISPATCHER_ENFORCE_SECURE"`
```

Comment:
@jirevwe We have a project-level configuration for this. I don't think we need it here as well.

Reply:
The dispatcher is created once and reused multiple times because the client and its transport need to be "created once and reused multiple times" so we can take advantage of connection reuse. If we configure it per project, we'd need to toggle it on and off at runtime, which might be messy.

Reply:
But we can't have both. We either remove it from the project settings or here. Another suggestion is to reuse the client and transport on multiple dispatcher instances.